Assets/LeapMotionModules/Interaction/InteractionBehaviourBase.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Leap;
+using Leap.Unity.Attributes;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.UI.Interaction {
+
+  public abstract class InteractionBehaviourBase : MonoBehaviour {
+
+    public InteractionManager interactionManager;
+
+    protected Rigidbody _body;
+    public new Rigidbody rigidbody { get { return _body; } }
+
+    [Header("Interaction Settings")]
+    public bool allowHover = true;
```

Comment:
Well the reason they were enableHover in the manager and ignoreHover originally was to emphasize that they are two different places in which you can ignore hover logic. You can have the manager ignore all hover logic, or you can override the manager's setting on a per-object basis. If they were named the same, it would be even more confusing. Come to think of it, I need to create a warning if allowHover/Contact/Grasp is checked but the corresponding enableX bool is not checked in the Manager, because the one does not override the other